### PR TITLE
feat: Image component implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

File: Components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+.responsive-image {
+    min-width: 100%;
+    height: auto;
+}
+.image-rd {
```
Comment (parvez-rashid) [inline Components/Image/image.css:6]: Give the class a more descriptive name like `.image-round`. Much easier to read and understand.

**Review by parvez-rashid — APPROVED** [review]
Please include a live preview of the app. All the best!

**Review by aurobindoGupta — APPROVED** [review]

**Review by saicharan-gouru — APPROVED** [review]
Looks good to me.

File: Components/Image/image.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Image/image.css:6]: okay, Thanks

**Pull request merged**